### DISCUSSION TITLE
Broken site when deploying with pages

### DISCUSSION THREAD

**Discussion by colebrazeal** [community/community#54370] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Here is the link to my site and Repo: 

https://yournamellc.github.io/anime-quote-generator/

https://github.com/yournamellc/anime-quote-generator

When I deploy with pages my site breaks and looks terrible and only the homepage works but all my css etc. isn't applying lol. Not sure what I'm doing wrong but I would love if someone could help me with fixing this.

**Comment by airtower-luna** [comment]
Your site is deployed under `/anime-quote-generator/`, but your links point at `/docs/` (absolute path), that's why it breaks. If you use relative links it'll work no matter the deployment path.

E.g. [here](https://github.com/yournamellc/anime-quote-generator/blob/de319f33b533bb8fa3176e27bff6581663b9c26b/docs/index.html#L6):

```html
        <link href="/docs/styles.css" rel="stylesheet" type="text/css" />
```

I assume you mean the `styles.css` that exists in the same directory, so it should be:

```html
        <link href="styles.css" rel="stylesheet" type="text/css" />
```

See also: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls
